code-quality/test-files-indentation.s.md

```diff
@@ -0,0 +1,70 @@
+# [Standard] Test files indentation
+
+## Owner: [Pierre-Louis Le Portz](https://github.com/pleportz)
+
+## Why
+
+Standardizing the indentation of a test file saves the developer 30 seconds per new function to test.
+
+## Checks
+
+The test file must have the following indentation even if only one function is tested:
+
+``` javascript
+describe('Name of the tested file', () => {
+  describe('function1', () => {
+    it('should do something', () => {
```

Comment:
`it` descriptions must not contain 'should'.
You wouldn't say out loud "I'm testing that Name of the tested file should do something", but "I'm testing that Name of the tested file does something"

See Jest's official doc for example: https://facebook.github.io/jest/docs/en/api.html#testname-fn-timeout

Reply:
👍